comments/stakeholder_annotations.md

```diff
@@ -96,7 +92,28 @@ The painpoint annotation data goes into the UserMapAnnotationBag (a database tab
 
 ### Output data
 
-An API is called which aggregates all of the UserMapAnnotationBags for the target map together, and returns a tabular data file similar to the one outlined in the objective. 
+An API is called which aggregates all of the UserMapAnnotationBags for the target map together. API calls are formatted as follows:
+
+    https://staging.ecosystem.guide/api/annotations/report?map=<guest_map_path>
+
+where `guest_map_path` is the guest map path property for the target map (can be viewed in the Admin panel).
```

Comment:
As noted in api draft PR, I think it's better to stick with IRI here and support qname, which has been implemented.